ResidenceManager/User.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import Firebase
 
 class User {
```

Comment:
Can you extend `Firebase.User` to create our `User` class so that we can handle it in simple way.